jp/components/themes/theming-demo.md

```diff
@@ -0,0 +1,335 @@
+</div>
+<div>
+    <button data-localize="stackblitz" class="stackblitz-btn" data-iframe-id="default-theme-sample-iframe" data-demos-base-url="{environment:demosBaseUrl}"> stackblitz �ŊJ��
+    </button>
+</div>
+<div class="divider--half"></div>
+
+�R���|�[�l���g�̊O�ς�ύX�������ꍇ��**�f�t�H���g �e�[�}**���A�v���P�[�V������ UI �ɍ���Ȃ��ꍇ�́A**Ignite UI for Angular Theming** ��g�p���đ����� CSS �t�@�C��������ɃX�^�C���ݒ肪�ȒP�ɂł��܂��B
+
+### �J�n
+
+�͂��߂ɁA**SASS functions �� mixins** �����q�ɂȂ�Ă���**�e�[�} ���[�e�B���e�B**��C���|�[�g���܂��B
+���z�I�ȃR�[�h�\���́A�ʁX�̃f�B���N�g��**�e�[�} ���W�b�N**�ɔz�u���邱�Ƃł��B
+
+```scss
+// styles/dark-theme.scss
+
+@import '~igniteui-angular/lib/core/styles/themes/utilities';
+
+```
+<div class="divider--half"></div>
+
+���̃X�e�b�v�́A�J�X�^�}�C�Y����R���|�[�l���g�ƑΉ�����e�[�}��ׂăC���|�[�g���܂��B
+
+�ȉ����A�v���Ɋ܂܂�܂��B
+- **Igx-Grid** �� `paging` �� **�t�B���^�����O**�B
+
+```scss
+// Import grid component and theme styles
+@import '~igniteui-angular/lib/core/styles/components/grid/grid-component';
+@import '~igniteui-angular/lib/core/styles//components/grid/grid-theme';
+
+// Import grid-paginator component and theme styles
+@import '~igniteui-angular/lib/core/styles//components/grid-paginator/grid-paginator-component';
+@import '~igniteui-angular/lib/core/styles//components/grid-paginator/grid-paginator-theme';
+
+// Import grid-filtering component and theme styles
+@import '~igniteui-angular/lib/core/styles//components/grid-filtering/grid-filtering-component';
+@import '~igniteui-angular/lib/core/styles//components/grid-filtering/grid-filtering-theme';
+
+```
+- **Igx-Dialog** �̑g�ݍ��� `IgxInputGroupComponent`�B
+
+```scss
+
+// Import dialog-group component and theme styles
+@import '~igniteui-angular/lib/core/styles//components/dialog/dialog-component';
+@import '~igniteui-angular/lib/core/styles//components/dialog/dialog-theme';
+
+// Import input-group component and theme styles
+@import '~igniteui-angular/lib/core/styles//components/input/input-group-component';
+@import '~igniteui-angular/lib/core/styles//components/input/input-group-theme';
+
+```
+
+�R���|�[�l���g�̃e�[�}�ݒ�ɕK�v�Ȃ��ׂĂ̋@�\�����邽�߁A�g�p����**�F�p���b�g**���`���邾���ł��B
+
+```scss
+// Define the primary color
+$dark-color: #282828;
+
+// Define the secondary color
+$orange-color: #FFA500;
+
+// Define the palette:
+$dark-theme-palette: igx-palette($primary: $dark-color, $secondary: $orange-color);
+```
+
+���ɃA�v���ɐV�����X�^�C����ݒ肵�܂��B
+<div class="divider--half"></div>
+
+### �e�[�}�̐ݒ�
+
+**�e�[�}�̐ݒ�**�́A�v���C�}������уZ�J���_���̐F�̈قȂ�V�F�[�h��R���|�[�l���g �p�[�c�ɓK�p���܂��B[**Ignite UI for Angular Themes**](https://staging.infragistics.local/products/ignite-ui-angular/docs/sass/) �̃w���v�ŃX�^�C���ݒ�\�ȃR���|�[�l���g �p�[�c��m�F�ł��܂��B
```

Comment:
This link supposedly isn't public yet.  We don't have a Japanese version.  What should we do?  There are other instances of these API links too.  Keep them in English for now and update them when we're ready?  Or will this not go out any time soon?